www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -150,3 +152,61 @@
 
     throw new \Exception($versionOfTheUpgrade . $errorMessage, (int)$e->getCode(), $e);
 }
+
+/**
+ * Insert SSO configuration
+ *
+ * @param CentreonDB $pearDB
+ * @return void
+ */
+function insertWebSSOConfiguration(CentreonDB $pearDB): void
+{
+    $pearDB->beginTransaction();
```

Comment:
Do not put the transaction in this function and include other previous SQL queries that may be transactional.